config/database.yml

```
@@ -11,6 +11,7 @@ default: &default
   password: "<%= ENV['MYSQL_PASSWORD'] || 'password' %>"
   host: "<%= ENV['MYSQL_HOST'] || '127.0.0.1' %>"
   encoding: utf8mb4
+  pool: 10
```

```suggestion
  pool: "<%= ENV['MYSQL_CONNECTION_POOL'] || '10' %>"
```

インスタンス数増えるとRDSのmax_connectionsに当たるリスクもあるのでconfigurableにしたほうがいいかなと

---

これはENVで指定できるようにした方が安全ですな。